ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -118,6 +120,11 @@ def _write_training_metrics(self):
             if brain_name in self.trainer_metrics:
                 self.trainers[brain_name].write_training_metrics()
 
+    def _write_timing_tree(self) -> None:
+        timing_path = f"{self.summaries_dir}/{self.run_id}_timers.json"
```

Comment:
I'm open to suggestions on where to put this/what to call it. But this will put it in the same path as the current TrainerMetrics csv files.